src/components/UserRegister.tsx

```diff
@@ -135,115 +65,144 @@ const UserForm = (props: any) => {
   };
```

Comment:
you don't have to use phonePattern state for any purpose
you can use it like this
`
if(Flags[index].code === '+1-United States'){
    setValue('country', 'US')
} else{
    ...
}
`

Reply:
also use enums